Gemfile

```diff
@@ -12,7 +12,7 @@ gem 'puma', '~> 4.1'
 # Use SCSS for stylesheets
 gem 'sass-rails', '>= 6'
 # Transpile app-like JavaScript. Read more: https://github.com/rails/webpacker
-gem 'webpacker', '~> 4.0'
+gem 'webpacker', '~> 5.0'
```

Comment:
How come you needed to update this? Also an update like this would be better in its own branch/PR

Reply:
oh this was updated in a small experiment. will put this back to `4.0`